Cargo.toml

```diff
@@ -18,7 +18,6 @@ base64 = { workspace = true }
 base-x = "0.2.11"
```

Comment:
The removal of memmap isn't there, but I think it doesn't matter as #777 is about to re-introduce it.